src/excmds.ts

```diff
@@ -1428,14 +1428,16 @@ export function scrollpage(n = 1, count = 1) {
  *  Rudimentary find mode, left unbound by default as we don't currently support `incsearch`. Suggested binds:
  *
  *      bind / fillcmdline find
- *      bind ? fillcmdline find -?
- *      bind n findnext 1
- *      bind N findnext -1
+ *      bind ? fillcmdline find --reverse
+ *      bind n findnext --search-from-view
+ *      bind N findnext --search-from-view --reverse
+ *      bind gn findnext
+ *      bind gN findnext --reverse
```

Comment:
`g{n,N}` in Vim starts visual mode on the next match, it would be nice to preserve those bindings for #4373

I don't know if we need to include a suggested bind for these.

Reply:
Just remove it if you want to.